cds_ils/migrator/eitems/api.py

```diff
@@ -344,3 +354,35 @@ def migrate_ebl_links(raise_exceptions=True):
             else:
                 if raise_exceptions:
                     raise exc
+
+
+def migrate_safari_links(raise_exceptions=True):
+    """Migrate Safari links from documents."""
+    document_class = current_app_ils.document_record_cls
+
+    search = get_documents_with_safari_eitems()
+    click.echo("Found {} documents with safari links.".format(search.count()))
+
+    for hit in search.scan():
+        # make sure the document is in DB not only ES
+        document = document_class.get_record_by_pid(hit.pid)
+        click.echo("Processing document {}...".format(document["pid"]))
+
+        try:
+            for item in document["_migration"]["eitems_safari"]:
+                eitem = create_eitem(document["pid"], open_access=False)
```

Comment:
if you set here `open_access= False` why did you add it as a property in the urls in the migration rules?

Reply:
I thought it would be more consistent if all the urls had the `open_access` property even if it's not used, but you are right, it doesn't make sense for Safari and EBL since they are always restricted. I will remove the property, thanks.